### PR TITLE
Loader syntax and Loader.load test

### PR DESCRIPTION
**Pull request by SPalominos**
Improve the syntax in Loader.groovy.
Add tests for Loader.load() process.

### CODE REVIEW

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy [hunk 1]
```diff
@@ -125,59 +116,56 @@ IProcess fromPlace() {
         title "Extract the OSM data using a place name"
         inputs datasource: JdbcDataSource, placeName: String, distance : 0
         outputs zoneTableName: String, zoneEnvelopeTableName: String, osmTablesPrefix: String, epsg:int
-        run { datasource,placeName,distance ->
-            String formatedPlaceName = placeName.trim().split("\\s*(,|\\s)\\s*").join("_");
-            def outputZoneTable = "ZONE_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
-            def outputZoneEnvelopeTable = "ZONE_ENVELOPE_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
-            def osmTablesPrefix = "OSM_DATA_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
+        run { JdbcDataSource datasource, placeName, distance ->
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/Loader.groovy:121]: JdbcDataSource not necessary

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy [hunk 2]
```diff
@@ -228,61 +213,34 @@ IProcess load() {
...
-                    datasource.load(osmFile, osmTablesPrefix, true)
-                    info "The input OSM file has been loaded in the database"
-                } else {
-                    error "The input OSM file does not exist"
-                    return null
-                }
-            } else {
-                error "Please set a valid database connection"
-                return null
+        run { JdbcDataSource datasource, osmTablesPrefix, osmFilePath ->
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/Loader.groovy:216]: idem

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy [hunk 3]
```diff
@@ -228,61 +213,34 @@ IProcess load() {
...
+            if(!datasource) {
+                error "Please set a valid database connection."
+                return
             }
-            [datasource: datasource]
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/osm/Loader.groovy:246]: removed ?

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy (hunk 1 above)
Comment (SPalominos) [reply osm/src/main/groovy/org/orbisgis/osm/Loader.groovy:121]: Right, but without, the IDE doesn't detect well the completion, do I remove it ?

File: osm/src/main/groovy/org/orbisgis/osm/Loader.groovy (hunk 3 above)
Comment (SPalominos) [reply osm/src/main/groovy/org/orbisgis/osm/Loader.groovy:246]: No, it's git which doesn't show well the code modification. Checkout a little further, there is the rest of the function.

**Comment by SPalominos** [comment]
Ready to be merged, if someone have any comment ?

**Comment by ebocher** [comment]
Go

**Pull request merged**